packages/toolkit/src/view/catalog/components/CloneCatalogDialog.tsx

```diff
@@ -78,9 +101,55 @@ export const CloneCatalogDialog = ({
   const { formState, reset, watch, setValue } = form;
   const nameValue = watch("name");
   const description = watch("description");
+  const selectedNamespace = watch("namespaceId");
+  const [namespaceType, setNamespaceType] =
+    React.useState<Nullable<"user" | "organization">>(null);
 
   const formattedName = formatName(nameValue);
 
+  React.useEffect(() => {
+    const getNamespaceType = async () => {
+      if (selectedNamespace && accessToken) {
+        const type = await checkNamespaceType(selectedNamespace, accessToken);
+        setNamespaceType(type);
+      } else {
+        setNamespaceType(null);
+      }
+    };
+
+    getNamespaceType();
+  }, [selectedNamespace, accessToken]);
+
+  const catalogs = useGetCatalogs({
+    accessToken,
+    ownerId: selectedNamespace,
+    enabled: enabledQuery && !!selectedNamespace,
+  });
+
+  const userSub = useAuthenticatedUserSubscription({
+    enabled: enabledQuery,
+    accessToken,
+  });
+
+  const orgSub = useOrganizationSubscription({
+    organizationID: selectedNamespace ? selectedNamespace : null,
+    accessToken,
+    enabled: enabledQuery && namespaceType === "organization",
+  });
+
+  const subscriptionInfo = React.useMemo(() => {
+    return getSubscriptionInfo(
+      namespaceType,
+      userSub.data || null,
+      orgSub.data || null,
+    );
+  }, [selectedNamespace, userSub.data, orgSub.data, namespaceType]);
```

Comment:
@thewbuk This deps has an unnecessary deps `selectedNamespace`, If your hook didn't work without this variables, then you might need to rethink a bit this useMemo usage